{"sections":[],"hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/error"},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"text":"error","kind":"identifier"},{"text":": ","kind":"text"},{"text":"APIError","preciseIdentifier":"s:6OpenAI8APIErrorV","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","kind":"typeIdentifier"}]}],"kind":"declarations"}],"kind":"symbol","metadata":{"roleHeading":"Instance Property","role":"symbol","title":"error","modules":[{"name":"OpenAI"}],"fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"text":"error","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:6OpenAI8APIErrorV","kind":"typeIdentifier","text":"APIError"}],"externalID":"s:6OpenAI16APIErrorResponseV5errorAA0C0Vvp","symbolKind":"property"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/openai\/apierrorresponse\/error"]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"references":{"doc://OpenAI/documentation/OpenAI":{"title":"OpenAI","url":"\/documentation\/openai","kind":"symbol","role":"collection","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","type":"topic","abstract":[]},"doc://OpenAI/documentation/OpenAI/APIErrorResponse/error":{"url":"\/documentation\/openai\/apierrorresponse\/error","role":"symbol","title":"error","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/error","fragments":[{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"error"},{"kind":"text","text":": "},{"preciseIdentifier":"s:6OpenAI8APIErrorV","text":"APIError","kind":"typeIdentifier"}],"type":"topic","abstract":[],"kind":"symbol"},"doc://OpenAI/documentation/OpenAI/APIError":{"url":"\/documentation\/openai\/apierror","navigatorTitle":[{"text":"APIError","kind":"identifier"}],"kind":"symbol","type":"topic","abstract":[],"title":"APIError","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","role":"symbol","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"APIError","kind":"identifier"}]},"doc://OpenAI/documentation/OpenAI/APIErrorResponse":{"type":"topic","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse","navigatorTitle":[{"text":"APIErrorResponse","kind":"identifier"}],"abstract":[],"kind":"symbol","url":"\/documentation\/openai\/apierrorresponse","title":"APIErrorResponse","role":"symbol","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"text":"APIErrorResponse","kind":"identifier"}]}}}